GLib-2.0.gir

```diff
@@ -13884,7 +13858,7 @@ levels using g_log_set_handler() and g_log_set_fatal_mask().</doc>
       <member name="level_debug" value="128" c:identifier="G_LOG_LEVEL_DEBUG">
         <doc xml:space="preserve">log level for debug messages, see g_debug()</doc>
       </member>
-      <member name="level_mask" value="-4" c:identifier="G_LOG_LEVEL_MASK">
+      <member name="level_mask" value="4294967292" c:identifier="G_LOG_LEVEL_MASK">
```

Comment:
shouldn't this be fixed upstream?

Reply:
You can add `(type uint)` annotation for this IIRC

Reply:
And `(value 4294967292)`. That's done in GStreamer in many places. It also confuses the Python bindings if you don't do that, but I guess nobody is using `G_LOG_LEVEL_MASK` from Python.

Reply:
I think we have this problem with any flags type, we consider them u32 but in C anything in an enum has to be `int`. Maybe I will just have gir cast them to unsigned

Reply:
Yeah it looks like those annotations won't work on an enum constant, they only work on a `#define`